<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListeVaccinDialog</class>
 <widget class="QDialog" name="ListeVaccinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche par nom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_tri_DD">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri Par Date Dev</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSnom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableViewVaccins">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rechercherPays">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>RPays</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonRefresh">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonSupprimer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonModifier">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSpays">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche par pays</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rechercherNom">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>RNom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonAjouter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_tri_DPrem">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri Par Date Prem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonSupprimer_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Historique Stock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AjoutFromPDF">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajout From PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
